website/docs/r/hbase_instance.html.markdown

```diff
@@ -61,6 +61,10 @@ The following arguments are supported:
 * `auto_renew` - (Optional, ForceNew) `true`, `false`, System default to `false`, valid when pay_type = PrePaid.
 * `vswitch_id` - (Optional, ForceNew) if vswitch_id is not empty, that mean net_type = vpc and has a same region. if vswitch_id is empty, net_type_classic
 * `cold_storage_size` - (Optional, ForceNew) 0 or 0+. 0 means is_cold_storage = false. 0+ means is_cold_storage = true
+* `maintain_start_time` - (Optional) The start time of the operation and maintenance time period of the instance, in the format of HH:mmZ (UTC time).
```

Comment:
(Optional, Available in v1.71.2+), please add the same content in the description of `maintain_end_time`  and `deletion_protection `